build-test.xml

```diff
@@ -6836,6 +6836,14 @@ ANT_OPTS=${env.ANT_OPTS}</echo>
 					</fileset>
 				</chmod>
 
+				<property name="test.build.additional.osgi.dependencies" value="" />
+
+				<prepare-additional-osgi-jars
+					additional.dependencies="${test.build.additional.osgi.dependencies}"
+					osgi.dir="${app.server.parent.dir}/osgi"
+					repository.url="${sonatype.release.url}"
+				/>
+
```

Comment:
Since test.build.additional.osgi.dependencies will always contain an empty string, I'm assuming that you intended for prepare-additional-osgi-jars to never do anything until you complete other parts of the patcher job implementation?